src/Microsoft.Diagnostics.Monitoring.WebApi/Metrics/JsonCounterLogger.cs

```diff
@@ -127,6 +136,10 @@ protected override async Task SerializeAsync(ICounterPayload counter)
             //Some versions of .Net return invalid metric numbers. See https://github.com/dotnet/runtime/pull/46938
             writer.WriteNumber("value", double.IsNaN(value) ? 0.0 : value);
 
+            writer.WriteString("meterTags", meterTags);
+            writer.WriteString("instrumentTags", instrumentTags);
+            writer.WriteString("scopeHash", scopeHash);
```

Comment:
Is scope hash meaningful for the user to see? If not, I would remove this and allow us to add it back later in some useful form in a future update.

Reply:
As of right now, I don't believe so - in which case, I think this makes sense (better to omit and add later, than to add it and want to remove it later as a breaking change).